.github/workflows/build.yml

```diff
@@ -148,6 +148,10 @@ jobs:
       run: npm install -g grunt-cli
     - name: Horti Install
       run: npm install -g horticulturalist
+    - name: Medic Conf Install
+      run: npm install -g medic-conf
```

Comment:
Can you pin the version of `medic-conf` as we start releasing more the actions might change once it is released and cause a problem.